st2common/st2common/cmd/purge_task_executions.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2019 Extreme Networks, Inc.
```

Comment:
Copyright is now StackStorm Authors, and year file first introduced, e.g. could you change to:

# Copyright 2021, The StackStorm Authors.